test/testkit/mocks/backend/config_map.go

```diff
@@ -73,11 +77,13 @@ service:
 
 const LogConfigTemplate = `receivers:
   fluentforward:
-    endpoint: 0.0.0.0:8006
+    endpoint: ${MY_POD_IP}:8006
```

Comment:
```suggestion
    endpoint: 127.0.0.1:8006
```
Try this reference: https://docs.splunk.com/observability/en/gdi/opentelemetry/components/fluentd-receiver.html